lib/css_parser/rule_set.rb

```diff
@@ -22,6 +22,30 @@ class RuleSet
       ['border-width', %w[border-top-width border-right-width border-bottom-width border-left-width]]
     ].freeze
 
+    class DeclarationValue
```

Comment:
is `Declaration` already taken ? ... because having `@declarations` include `Declaration` would be nice :)